src/components/toast/toast.module.scss

```diff
@@ -0,0 +1,79 @@
+@import '../../styles/common/variables.module.scss';
+.root {
+    width: 350px;
+    border-radius: 6px;
+    padding: 15px;
+    display: grid;
+    grid-template-rows: auto auto;
+    grid-template-columns: 1fr auto;
+    align-items: center;
+    column-gap: 10px;
+    background-color: $sand1;
+    border: 1px solid $sand2;
+    box-shadow:
+        0 10px 20px -15px rgba(14, 18, 22, 0.2),
+        0 10px 38px -10px rgba(13, 18, 22, 0.35);
+}
+.action {
+    grid-row: 1 / 3;
+    grid-column: 2 / 3;
+}
+.title {
+    font-size: 15px;
+    font-weight: 700;
+    grid-row: 1 / 2;
+    grid-column: 1 / 2;
+}
+.description {
+    font-size: 13px;
+    grid-row: 2 / 3;
+    grid-column: 1 / 2;
+}
+
+.root[data-state='open'] {
```

Comment:
I copied everything related to animation from Radix example, as there is nothing we can do in this regard in Codux.